LoopKitUI/Views/GuidePage.swift

```diff
@@ -37,8 +37,10 @@ public struct GuidePage<Content, ActionAreaContent>: View where Content: View, A
             VStack {
                 self.actionAreaContent
             }
+            .padding(self.horizontalSizeClass == .regular ? .bottom : [])
             .background(Color(UIColor.systemBackground).shadow(radius: 5))
         }
+        .edgesIgnoringSafeArea(.bottom)
```

Comment:
Does this affect scrolling and the navigation title? I remember when I needed to set the background and ignored the safe area, scrolling just stopped.

Reply:
In my testing, it doesn't, probably because this is on the non-scrolling area of the view.  I tested both the scrolling and the large title shrinking on scroll aspects.